File: bestia-zoneserver/src/test/java/net/bestia/zoneserver/map/path/PointEstimatorTest.java
package net.bestia.zoneserver.map.path;

import org.junit.Assert;
import org.junit.Test;

import bestia.model.geometry.Point;

public class PointEstimatorTest {
	
	@Test
	public void getDistance_twoPoints_euclidianDistance() {
		
		PointEstimator pe = new PointEstimator();
		
		Point p1 = new Point(1, 5);
		Point p2 = new Point(3, 10);
		
		float d = pe.getDistance(p1, p2);
		Assert.assertEquals(5.385, d, 0.01);
	}

}

File: bestia-zoneserver/src/test/kotlin/net/bestia/zoneserver/map/path/PointEstimatorTest.kt
package net.bestia.zoneserver.map.path

import org.junit.Assert
import org.junit.Test

import net.bestia.model.geometry.Point

class PointEstimatorTest {
  @Test
  fun getDistance_twoPoints_euclidianDistance() {
    val pe = PointEstimator()

    val p1 = Point(1, 5)
    val p2 = Point(3, 10)

    val d = pe.getDistance(p1, p2)
    Assert.assertEquals(5.385, d.toDouble(), 0.01)
  }
}
